app/assets/stylesheets/custom_css/event.css.scss

```diff
@@ -0,0 +1,231 @@
+.event {
```

Comment:
I see a lot of duplication in the CSS, you scope it page with a class at the top, but perhaps you should leave common classes outside that scope and reuse them